app/assets/stylesheets/components/_madlib-statement.scss

```diff
@@ -16,3 +16,27 @@
     margin-left: $base-spacing;
   }
 }
+
+.underline {
+  display: flex;
+  justify-content: space-between;
+}
+
+.field_with_errors {
+  .selectize-control .selectize-input.items {
+    background-color: tint($required-color, 90%);
+    border-bottom-color: $required-color;
+  }
+
+  .error {
+    display: block;
+
+    @at-root .error {
```

Comment:
It tells sass, "hey bump this up outside of the nested selector and make a rule." It's a way to keep things a little more organized. So, rather that having a `.error { display: none}` randomly in the sheet. Now, there's a logical relationship.